code/modules/uplink/uplink_item.dm

```diff
@@ -715,6 +715,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	item = /obj/item/grenade/clusterbuster/soap
 	cost = 6
 
+/datum/uplink_item/stealthy_weapons/rubberduck
+	name = "Druggy Ducky"
+	desc = "A seemingly innocent rubber ducky. Squeaking it will start a timer that will set off a chemical payload containing drugs. \
```

Comment:
`ducky` - > `duck`

Reply:
the rubber duck is referred to as "Rubber Ducky", so i am calling it "Druggy Ducky"